developer/code/license.md

```diff
@@ -12,7 +12,8 @@ title: Software license
 
 Qubes OS is a compilation of software packages, each under its own license. The
 compilation is made available under the GNU General Public License version 2
-(GPLv2).
+(GPLv2). However, the license for this compilation does not supersede the
+license of any package included in the compilation.
```

Comment:
I am not an expert on free software licensing, so I have no idea whether this helps or makes sense. However, @HW42 pointed out to me that the Fedora license says this:

> Fedora Linux is a compilation of software packages, each under its own licensing terms. The compilation itself is released under the MIT license. However, this compilation license does not supersede the licenses of code and content contained in Fedora Linux, which conform to the guidelines described at License Approval.
> 
> With respect to the notice inclusion provision of the MIT license, solely with respect to the Fedora Linux compilation license grant, Fedora does not regard non-Fedora-branded downstream derivative distributions as being "copies or substantial portions of the Software."

The first two sentences are already similar to what we have, but we don't have anything similar to the third sentence, even though we seem to be in a similar situation. (I have no idea about the second paragraph. It sounds like it may not apply to us.)

So, I've attempted to add something to cover what appears to be missing. Again, as a layman, I have no idea whether this is right or makes sense. This is just my humble attempt to help and make things better. If this addition doesn't do that, then please reject, remove, and discard it.